pkg/cli/deployment/deploy_test.go

```diff
@@ -54,8 +54,6 @@ func Test_GetProviderConfigs(t *testing.T) {
 func Test_GetProviderConfigsWithAzProvider(t *testing.T) {
 	resourceDeploymentClient := ResourceDeploymentClient{
 		RadiusResourceGroup: "testrg",
-		Client:              &sdkclients.ResourceDeploymentsClient{},
-		OperationsClient:    &sdkclients.ResourceDeploymentOperationsClient{},
```

Comment:
These weren't actually doing anything in the test. We should write tests using the interface like I do in the deploymenttemplate controller unit tests.